frontend/src/core/user/components/UserAvatar.js

```diff
@@ -13,30 +14,34 @@ export default function UserAvatar(props: Props) {
     const { user, username, title, imageUrl } = props;
 
     if (!user) {
-        return <div className='user-avatar'>
-            <img
-                src='/static/img/anon.jpg'
-                alt='Anonymous User'
-                height='44'
-                width='44'
-            />
-        </div>;
+        return <Localized id='user-avatar-anonymous' attrs={{ title: true }} >
+            <div className='user-avatar'>
+                <img
+                    src='/static/img/anon.jpg'
+                    alt='Anonymous User'
+                    height='44'
+                    width='44'
+                />
+            </div>;
+        </Localized>
     }
 
-    return <div className='user-avatar'>
-        <a
-            href={ `/contributors/${username}` }
-            title={ title }
-            target='_blank'
-            rel='noopener noreferrer'
-            onClick={ (e: SyntheticMouseEvent<>) => e.stopPropagation() }
-        >
-            <img
-                src={ imageUrl }
-                alt='User Profile'
-                height='44'
-                width='44'
-            />
-        </a>
-    </div>
+    return <Localized id='user-avatar' attrs={{ title: true }} >
```

Comment:
In this case, what you want to localize is the `alt` attribute of the `img` tag. This `Localized` component should thus be directly around the `<img>` tag, and you'll want to change the `attrs` prop to set `alt: true` instead.

Currently, what this is effectively doing is localizing the title of the `<a>` tag, but that title being passed from outside, you don't actually know what it contains and thus how it should be localized. It will be up to the components using `UserAvatar` to pass it a localized title attribute.